PublicMapping/districtbuilder
# Refactor evaluate mode

## Overview

- Move files for Project Evaluate mode into the `evaluate` folder
- Splits the `ProjectEvaluateSidebar` component into a wrapper component, and puts info display from sidebar into new `ProjectEvaluateSummary` component
- Displays computed County Split metric in `ProjectSummary` to show an example of the computation / flow of data within these components

### Checklist



## Testing Instructions

- ./scripts/server
- Prefer bulleted description
- Start after checking out this branch
- Include any setup required, such as bundling scripts, restarting services, etc.
- Include test case, and expected output
